lib/workers/repository/onboarding/branch/rebase.js

```diff
@@ -1,16 +1,19 @@
-async function rebaseOnboardingBranch() {
+const { getOnboardingConfig } = require('./config');
+
+async function rebaseOnboardingBranch(config) {
   logger.debug('Checking if onboarding branch needs rebasing');
+  let contents = await getOnboardingConfig(config);
+  const onboardingBranch = `renovate/configure`;
```

Comment:
Can this rely on the `branchName` configuration so that if a user of Renovate sets the appropriate environment variable to configure the branch name prefix, it will be used by the on-boarding PR? (This may be out of scope for this change since it's existing behavior)

Reply:
I thought about it and decided it would be too weird to raise an onboarding PR in `renovate/config` and then allow a user to define `branchPrefix` and delete/recreate the onboarding.

However it might be nice for some to allow the branch prefix to be configurable in `config.js` or env when in self-hosting mode, but this is currently not supported and more than just one line.

Reply:
> However it might be nice for some to allow the branch prefix to be configurable in config.js or env when in self-hosting mode, but this is currently not supported and more than just one line.

^ That was the specific use case I was thinking about. We don't need that specific feature in the instance of Renovate we run, but perhaps others will want to have the on-boarding PR conform to a naming convention.